src/connection-manager/utils.ts

```diff
@@ -0,0 +1,220 @@
+import WebSocket from 'ws'
+
+import {
+  query,
+  saveAmendmentEnabled,
+  saveAmendmentsEnabled,
+} from '../shared/database'
+import {
+  AmendmentEnabled,
+  DatabaseValidator,
+  Fee,
+  LedgerEnableAmendmentResponse,
+  LedgerEntryAmendmentsResponse,
+  StreamLedger,
+  StreamManifest,
+  TxEnableAmendmentResponse,
+  ValidationRaw,
+} from '../shared/types'
+import { rippleTimeToUnixTime } from '../shared/utils'
+
+import agreement from './agreement'
+import { handleManifest } from './manifests'
+
+const LEDGER_HASHES_SIZE = 10
+const AMENDMENT_LEDGER_ENTRY_INDEX =
```

Comment:
This value is exported by the `xrpl` package.  Maybe we move `xrpl` into `dependencies`.